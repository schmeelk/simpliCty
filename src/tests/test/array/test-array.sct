int main () {
   int x;
   int [11] n; 
   x = 999999;
   n[0] = 72 ;
   n[1] = 69 ;
   n[2] = 76 ;
   n[3] = 76 ;
   n[4] = 79 ;
   n[5] = 32 ;
   n[6] = 87 ;
   n[7] = 79 ;
   n[8] = 82 ;
   n[9] = 76 ;
   n[10] = 68 ;
   n[11] = 10 ;
   putchar (n[0]);
   putchar (n[1]);
   putchar (n[2]);
   putchar (n[3]);
   putchar (n[4]);
   putchar (n[5]);
   putchar (n[6]);
   putchar (n[7]);
   putchar (n[8]);
   putchar (n[9]);
   putchar (n[10]);
   putchar (n[11]);
   return 0;
}
