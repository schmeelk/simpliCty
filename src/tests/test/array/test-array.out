22
33
44
55
66
77
88
99
333
444
